operator/pkg/upgrade/upgrade_test.go

```diff
@@ -157,7 +158,7 @@ config:
 			},
 			validate: func(t *testing.T, updatedConfig *k0sv1beta1.ClusterConfig) {
 				// Verify that the unsupported override was applied to the telemetry config
-				assert.Equal(t, false, updatedConfig.Spec.Telemetry.Enabled)
+				assert.Equal(t, false, *updatedConfig.Spec.Telemetry.Enabled)
```

Comment:
would this compile in the 1.29 branch? i assume it wouldn't be a pointer and this wouldn't compile

Reply:
ah right and once we merge this in the release branch we're 💀 good call

Reply:
Fixed via - https://github.com/replicatedhq/embedded-cluster/pull/2148/commits/0c54c17835e7b39de8158842b34ecfe8c89aba7a